tests/unit/v2authTests.js

```diff
@@ -79,8 +78,20 @@ function handler(req, res) {
         const testCaseIndex = processRequest(
             JSON.parse(req.headers.additionaldata));
         if (typeof testCaseIndex === 'number') {
-            makeResponse(res, responseHeaders[testCaseIndex].code,
-                responseHeaders[testCaseIndex].message);
+            if (testCaseIndex === 0) {
+                makeResponse(res, responseHeaders[testCaseIndex].code,
+                    responseHeaders[testCaseIndex].message);
+            } else {
+                let code = 'Unknown';
```

Comment:
This is repeated from the previous file. Can you create a `testUtils.js` and put it in there?